casepro/backend/junebug.py

```diff
@@ -115,7 +188,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after, created_at__lte=modified_before)
```

Comment:
Oh that's interesting. Yes an AND is how the django filter arguments work, and it's what we want. Looking at this though, I thought that multiple arguments wouldn't work: https://github.com/praekelt/seed-identity-store/blob/develop/identities/views.py#L63